aws/v11.4.1/README.md

```diff
@@ -0,0 +1,74 @@
+# :zap: Giant Swarm Release v11.4.1 for AWS :zap:
+
+<< Add description here >>
```

Comment:
This release re-activates the recent AWS release of high-availability (HA) masters while fixing issues with certain OIDC configurations.